lib/chainComposer.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
Could you move this under a `utils` and export it like `module.exports.utils = require('./utils');`.
Would you mind adding a unit test?